spec/models/bot_spec.rb

```diff
@@ -9,6 +9,11 @@
     expect(bot).to be_valid
   end
 
+  it "defaults a bot's verify token" do
+    bot = Bot.create_prepared!(user)
+    expect(bot.channels.first.config["verify_token"]).to_not be_empty
+  end
```

Comment:
Isn't this test very specific for testing a bot creation on a facebook channel? If so, it could be better to only assert the first facebook channel of the bot. Just in case in the future the bot creates prepared for other channels.

Reply:
It's ok for now

Reply:
We will need to do some re-work for https://github.com/instedd/aida-ui/issues/150, so there is no point in over-thinking this right now.

Reply:
`Bot.create_prepared!` only creates Facebook bots for the time being. Tests should break whenever we change its behaviour - and that's OK